### PR TITLE
Resolve #445 Document new setting for the default value of click and …

### LINKED ISSUE #445: Document new CiviMail Tracking settings as per core PR #17291

### PR DESCRIPTION
**Pull request by seamuslee001**
…open tracking

also ping @MegaphoneJon @mfb

### CODE REVIEW

File: docs/email/mass-mailings-using-civimail.md [hunk 1]
```diff
@@ -269,6 +269,8 @@ two options described next.
 Once you have defined all aspects of your mailing click on **Next >**
 (lower left) to advance to:
 
+Prior to CiviCRM version 5.27 Tracking options were turned on by default when creating a new mailing. From CiviCRM 5.27 onwards there is now a new setting at Administer -> CiviMail -> CiviMail Component Settings which will allow you to choose whether the Click Through or the Open tracking or both are turned on by default or not.
```
Comment (mfb) [inline docs/email/mass-mailings-using-civimail.md]: This paragraph should be move up by one paragraph (so "Once you..." follows the new paragraph), and I would reword it slightly to say:
> Prior to CiviCRM version 5.27, tracking options were turned on by default when creating a new mailing. From CiviCRM 5.27 onwards, there are settings at Administer -> CiviMail -> CiviMail Component Settings which allow you to choose whether or not click-through and open tracking are turned on by default.

**Review by mfb — CHANGES_REQUESTED** [review]

**Review by homotechsual — APPROVED** [review]

File: docs/email/mass-mailings-using-civimail.md (hunk 1 above)
Comment (homotechsual) [reply docs/email/mass-mailings-using-civimail.md]: Actioned. Merging!
